Dockerfile

```diff
@@ -17,12 +19,149 @@ COPY --from=builder \
     /tmp/bitgo/modules/express/RELEASE_NOTES.md \
     /tmp/bitgo/modules/express/package.json \
     /var/bitgo-express/
-# copy both the root node_modules and the express module node_modules to the final node_modules directory
-COPY --from=builder /tmp/bitgo/node_modules /tmp/bitgo/modules/express/node_modules /var/bitgo-express/node_modules/
-# copy individual directories that contain js executable files, source and compiled js
+
+# copy the root node_modules to the bitgo-express parent node_modules
+COPY --from=builder /tmp/bitgo/node_modules  /var/node_modules/
+
+# copy the express module node_modules to the final node_modules directory
+COPY --from=builder /tmp/bitgo/modules/express/node_modules /var/bitgo-express/node_modules/
+
+# copy individual directories that contain js executable files, and compiled js
 COPY --from=builder /tmp/bitgo/modules/express/bin /var/bitgo-express/bin/
 COPY --from=builder /tmp/bitgo/modules/express/dist /var/bitgo-express/dist/
-COPY --from=builder /tmp/bitgo/modules/express/src /var/bitgo-express/src/
+
+# copy bitgo and yarn link it
+COPY --from=builder /tmp/bitgo/modules/bitgo/package.json /var/modules/bitgo/
+COPY --from=builder /tmp/bitgo/modules/bitgo/dist /var/modules/bitgo/dist/
+COPY --from=builder /tmp/bitgo/modules/bitgo/node_modules /var/modules/bitgo/node_modules/
+RUN cd /var/modules/bitgo && yarn link
+
+# copy bitgo dependencies
+# copy abstract-eth and yarn link it
+COPY --from=builder /tmp/bitgo/modules/abstract-eth/package.json /var/modules/abstract-eth/
+COPY --from=builder /tmp/bitgo/modules/abstract-eth/dist /var/modules/abstract-eth/dist/
+COPY --from=builder /tmp/bitgo/modules/abstract-eth/node_modules /var/modules/abstract-eth/node_modules/
+RUN cd /var/modules/abstract-eth && yarn link
+
+# copy abstract-utxo and yarn link it
+COPY --from=builder /tmp/bitgo/modules/abstract-utxo/package.json /var/modules/abstract-utxo/
+COPY --from=builder /tmp/bitgo/modules/abstract-utxo/dist /var/modules/abstract-utxo/dist/
+RUN cd /var/modules/abstract-utxo && yarn link
+
+# copy account-lib and yarn link it
+COPY --from=builder /tmp/bitgo/modules/account-lib/package.json /var/modules/account-lib/
+COPY --from=builder /tmp/bitgo/modules/account-lib/dist /var/modules/account-lib/dist/
+COPY --from=builder /tmp/bitgo/modules/account-lib/node_modules /var/modules/account-lib/node_modules/
+RUN cd /var/modules/account-lib && yarn link
+
+# copy blake2b and yarn link it
+COPY --from=builder /tmp/bitgo/modules/blake2b/package.json /var/modules/blake2b/
+COPY --from=builder /tmp/bitgo/modules/blake2b/index.js /var/modules/blake2b/
+COPY --from=builder /tmp/bitgo/modules/blake2b/node_modules /var/modules/blake2b/node_modules/
+RUN cd /var/modules/blake2b && yarn link
+
+# copy blake2b-wasm and yarn link it
+COPY --from=builder /tmp/bitgo/modules/blake2b-wasm/package.json /var/modules/blake2b-wasm/
```

Comment:
is there anything in blake2b-wasm we _aren't_ copying over?

Reply:
Just `example.html`, `example.js`, and `test.js` aren't copied over.